app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Should get a vaccine first")
+
+        vaccine_info = visitor["vaccine"]
+        expiration_date = vaccine_info.get("expiration_date")
+        if expiration_date < datetime.date.today():
```

Comment:
The comparison `if expiration_date < datetime.date.today():` assumes that `expiration_date` is a `datetime.date` object. Ensure that `expiration_date` is indeed a `datetime.date` object before this comparison. If `expiration_date` is a string or another type, you need to convert it to a `datetime.date` object first.